index.html

```diff
@@ -0,0 +1,203 @@
+              <li><a class="nav-list" href="#">Services</a></li>
+              <li><a class="nav-list" href="#">Contact</a></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <!-- main section -->
+      <section class="home-page fonts">
+        <p class="hello">"Impossible is nothing!"</p>
+        <h1 class="web-title fonts-coco">
+          RMsoft.ltd - Reliable Made Software
+        </h1>
+        <div class="h-content">
+          <p>
+            RMsoft.ltd company (Reliable Made Software) founded in 2012 by a
+            team of professionals who are passionate about delivering the best
+            web based software and android applications
+          </p>
+        </div>
+        <h2 class="dates fonts-coco">Impossible is nothing</h2>
+        <p class="h-copyright">
+          Improving people’s work through reliable softwares necessary in their
+          activities
+        </p>
+      </section>
+    </main>
+    <!-- Main program -->
+    <section class="main-program fonts">
+      <div class="p-title">
+        <h2>SERVICES</h2>
+        <hr class="line" />
+      </div>
+
+      <div class="programs">
+        <!-- program 1 -->
+        <div class="program">
+          <div class="p-header">
+            <i class="fa services fa-desktop"></i>
+            <h4>Website design</h4>
+          </div>
+          <p class="p-text">
+            You need more than just a website. You need results. We turn
+            underperforming websites around. We have the business acumen,
+            creativity, and experience
+          </p>
+        </div>
+        <!-- program 2 -->
+        <div class="program">
+          <div class="p-header">
+            <i class="fa services fa-code"></i>
+            <h4>Software</h4>
+          </div>
+          <p class="p-text">
+            Our softwares keeps us accountable. We pride ourselves on
+            responsibly taking our team can deliver on. We strive to not only
+            meet our clients’ expectations but exceed them
+          </p>
+        </div>
+        <div class="program">
+          <div class="p-header">
+            <i class="fa services fa-graduation-cap"></i>
+            <h4>Events</h4>
+          </div>
+          <p class="p-text">
+            We study the brand, identifying its target audience, devising the
+            event concept, and coordinating the technical aspects before
+            actually launching the event and provide online registration systems
+          </p>
+        </div>
+        <div class="program">
+          <div class="p-header">
+            <i class="fa services fa-file-video-o"></i>
+            <h4>Access control</h4>
+          </div>
+          <p class="p-text">
+            We provide High-End ultra series Quality Camera installation with
+            the highest features for enterprise and critical locations that
+            require a versatile and innovative solution to accommodate the site
```

Comment:
- [ ] Kindly ensure that the boxes are five, like it is in the design, as this is also a requirement 😉.